### PR TITLE
feat(Nitro Type): update presence

### LINKED ISSUE #8632: Nitro Type | nitrotype.com - add user placements in leagues
### Presence name

Nitro Type

### Description

in nitrotype.com/leagues, there are user standings and team standings. I would to see the user's placements in their league through the presence. This would be nice as an added detail to the /leagues presence. For example, we already have race stats.

### Alternatives

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by joerkig**
## Description 

Updated various parts of the activity due to site changes
Added showing users selected car

Resolves #8632 

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image
image
image

</details>

### CODE REVIEW

**Comment by joerkig** [comment]
ugh messed up a commit one moment

**Comment by joerkig** [comment]
> ugh messed up a commit one moment

Okay to review now

**Comment by joerkig** [comment]
Actually if you give a bit I'll more thoroughly add #8632

File: websites/N/Nitro Type/presence.ts [hunk 1]
```diff
@@ -1,121 +1,299 @@
...
+			const selectedShop = document.querySelector(
+				".page-shop--dealership-option.is-selected"
+			);
+			!selectedShop.classList.contains("season_shop")
+				? (presenceData.state = selectedShop
+						.querySelector("img")
+						.alt.replace("Level 1", "Loh's Cars")
+						.replace("Level 2", "Midtown Motors")
+						.replace("Level 3", "Hai Auto"))
+				: (presenceData.state = "Season Shop");
```
Comment (Bas950) [inline websites/N/Nitro Type/presence.ts:264]: make things like this a normal if case, would be easier to read.

**Review by Bas950 — APPROVED** [review]
For the rest LGTM

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**